vogue/adapter/plugin.py

```diff
@@ -36,6 +36,23 @@ def add_or_update_sample(self, sample_news: dict):
         else:
             LOG.info(f"No updates for sample {lims_id}.")
 
+    def add_or_update_application_tag(self, application_tag_news: dict):
+        """Adds/updates a application_tag in the database"""
+
+        tag = application_tag_news['_id']
+        update_result = self.db.application_tag.update_one({'_id' : tag}, {'$set': application_tag_news}, upsert=True)
+
+        if not update_result.raw_result['updatedExisting']:
+            self.db.application_tag.update_one({'_id' : tag}, 
+                {'$set': {'added': dt.today()}})
+            LOG.info(f"Added application_tag {tag}.")
```

Comment:
Depending on the log level, logging might not be executed, so why interpolate the variable beforehand? ;)

```suggestion
            LOG.info("Added application_tag %s.", tag)
```

Reply:
Someone (@moonso or @hassanfa ?) told me one should use the format I use (with %s) when making log strings because they are "faster". Thats why I don't use f"{}".

Reply:
I agree with @ingkebil , it should be `LOG.info("Added application_tag %s.", tag)`